jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -65,13 +68,30 @@ class AccordionSection extends React.Component {
   }
 
   /**
+    The callback for after the content has been mounted/unmounted.
+    @param content {Object} The content element that was (un)mounted.
+  */
+  _contentRefCallback(content) {
+    this['accordion-section-content'] = content;
```

Comment:
Interesting approach.

Is there a reason why you did this instead of doing it in the `componentDidMount` method?

Reply:
No, I don't think so. I think I only did it here as the callback was already there, but `componentDidMount` is probably a better place. I'll update it.